tests/scenario/migrated/asset-controller.test.ts

```diff
@@ -0,0 +1,244 @@
+import { beforeEachTruncateCollections } from "../../hooks/collections";
+import { beforeAllCreateEngines } from "../../hooks/engines";
+import { beforeEachLoadFixtures } from "../../hooks/fixtures";
+
+import { useSdk, sendPayloads } from "../../helpers";
+
+jest.setTimeout(10000);
+
+describe("features/Asset/Controller", () => {
+  const sdk = useSdk();
+
+  beforeAll(async () => {
+    await sdk.connect();
+    await beforeAllCreateEngines(sdk);
+  });
+
+  beforeEach(async () => {
+    await beforeEachTruncateCollections(sdk);
+    await beforeEachLoadFixtures(sdk);
+  });
+
+  afterAll(async () => {
+    sdk.disconnect();
+  });
```

Comment:
You can improve the generation framework by generate all of that part (generated by   writeFileBeginning ()) in one function inside a separated file imported in each generated file.
As I can see, it's not dependant of the test file and never change.

Reply:
Unfortunately I don't have the framework anymore to modify everything again and anyway those tests shouldn't move so I think we can leave it as is